### ISSUE TITLE
Remove mev-geth

### ISSUE THREAD

**Issue by selfuryon** [nix-community/ethereum.nix#166]
Do we need `mev-geth`? It seems like unsupported and dunno how to use that after the merge and mev-boost implementation.

**Comment by brianmcgee** [comment]
What makes you think it's no longer supported? I haven't been paying close enough attention to developments in that space over the last few months so I don't feel I have the right context to decide if it should be removed or not.

**Comment by selfuryon** [comment]
Maybe I don't understand what is `mev-geth`, but I thought that it's was a first mev realization by flashbots for eth1 before merge. They create a geth version which can get block from flashbot infrastructure.  After the merge we have PoS and ethereum implement PBS for creating `execution_payload` separately from local builder, and flashbot create `mev-boost`, relays and so on. So currently (I thought) we can't use `mev-geth` because mev is realized via `bulder-api` and mev-boost. 

But honestly, I haven't any experience with `mev-geth` and that area, only with mev-boost after the merge.
I also checked they repository and found that they haven't any recent commits in master, that's why I decided that it's unsupport and unneeded now :) But yeah, we can leave it as is.
